server/datastore/mysql/software_installers.go

```diff
@@ -191,12 +191,43 @@ func (ds *Datastore) DeleteSoftwareInstaller(ctx context.Context, id uint) error
 	return nil
 }
 
-func (ds *Datastore) InsertSoftwareInstallRequest(ctx context.Context, hostID uint, softwareTitleID uint, teamID *uint) error {
+func (ds *Datastore) GetSoftwareInstallerForTitle(ctx context.Context, softwareTitleID uint, teamID *uint) (*fleet.SoftwareInstaller, error) {
```

Comment:
I think we'll have a couple of those, Sarah implemented one in her PR too :) Maybe it's easier to dedupe this once it's all in the feature branch?

Reply:
lol, thanks for catching this. I agree and added an item to the cleanup task